types/nodejs-captcha/nodejs-captcha-tests.ts

```diff
@@ -0,0 +1,9 @@
+import captcha = require('nodejs-captcha');
+
+const testConfig = {
+	length: 8,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/nodejs-captcha/nodejs-captcha-tests.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.